### PR TITLE
Update dockerhub setup

### PR DESCRIPTION
**Pull request by justinharringa**
The previous creds were expired for the Docker Hub security incident. New cred time.

### CODE REVIEW

File: .travis.deploy.sh [hunk 1]
```diff
@@ -18,5 +18,5 @@ cp .travis.settings.xml ${HOME}/.m2/settings.xml
 mvn releaser:release
 
 # Push latest docker image
-docker login -u="${DOCKER_USER}" -p="${DOCKER_PASSWORD}"
+echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
```
Comment (npwolf) [inline .travis.deploy.sh]: For safety, do you want to wrap this in a set +x echo set -x . Guessing travis probably hides this anyway, but just a thought.

Comment (afalko) [reply .travis.deploy.sh]: why remove curlies?

Comment (justinharringa) [reply .travis.deploy.sh]: Secrets are hidden in Travis

Comment (justinharringa) [reply .travis.deploy.sh]: I'll add curlies for curliness without running`curl` :smile:

Comment (justinharringa) [reply .travis.deploy.sh]: I'll wrap it cuz why not

Comment (justinharringa) [reply .travis.deploy.sh]: `curly` `+x-x` FTW

**Pull request merged**